### PR TITLE
Update maintainers based on historical information

### PR DESCRIPTION
**Pull request by zachthompson**
Script to update the maintainer field for live instant answers based on historical data.

### CODE REVIEW

**Comment by MariagraziaAlastra** [comment]
What about giving them edit permissions for their IA Page, too?

**Comment by zachthompson** [comment]
Is that some flag we need to set in DB or...?  I didn't have that in the scope of this task.

**Comment by MariagraziaAlastra** [comment]
<pre>$ia->add_to_users($user)</pre>
Like it's done [here](https://github.com/duckduckgo/community-platform/blob/master/script/ghIssues.pl#L579), but I can add that, nbd

**Comment by zachthompson** [comment]
Ok, let me check it out. I can do that.

**Comment by zachthompson** [comment]
@MariagraziaAlastra  Added `add_to_user`.  Seems to work.

**Comment by zachthompson** [comment]
Got it, thanks!

File: sql/1.152/maintainer_update.pl [hunk 1]
```diff
@@ -0,0 +1,768 @@
...
+			$maintainer = qq|{"github":"$git_login"}|;
+		}
+
+		$ia->update({maintainer => $maintainer});
+	}
+	else{ warn "Failed to lookup IA with id of $id" }
+}
+
+__DATA__
+us_government_agencies_cheat_sheet	aasoliz
```
Comment (MariagraziaAlastra) [inline sql/1.152/maintainer_update.pl]: These are all github usernames, right?

Comment (zachthompson) [inline sql/1.152/maintainer_update.pl]: Yep.

**Comment by MariagraziaAlastra** [comment]
Tested and LGTM, thanks! :+1:

**Pull request merged**